[jenkins] Improve failure reporting slightly by listing 'Test docs' and 'Test run' as failing stages in the commit comment.

rolfbjarne opened the pull request.

monojenkins commented:
Build success
ℹ️ [Skipped execution](https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/8700//console)

rolfbjarne commented:
Test failures are unrelated:

* Package XM tests: fixed with PR #5149.
* Test docs: https://github.com/xamarin/maccore/issues/1118
* Test run: fixed with PR #5149.
